src/test/analyzer_test.ts

```diff
@@ -231,14 +233,14 @@ suite('Analyzer', () => {
     const analyzer = new BuildAnalyzer(config);
 
     // Check that data isn't flowing through sources until consumer usage
-    assert.isNull(analyzer.sources._readableState.flowing);
+    assert.isNull((<any>analyzer.sources)._readableState.flowing);
```

Comment:
Good catch, I should add a comment here. 

So the official Node TS definition doesn't define the `_readableState` property on streams because it is private. However, the official Node.js documentation documents this property [extensively](https://nodejs.org/api/stream.html#stream_three_states), and specifically as a way to check whether a stream is flowing or not. So we're stuck with TS telling us the property doesn't exist, and Node.js docs telling us it does and should be used in this situation.

We could also solve this by extending the Node typedef in `custom_typings/`. But since we only need it in one or two tests, scoping the change here with `<any>` seemed more appropriate.